yaksh/templates/yaksh/user_data.html

```diff
@@ -83,7 +84,7 @@
             {%endif%}
         </div>
     </div>
-    {% if question.type == "mcq" or question.type == "mcc" %}
+    {% if question.type != "code" %}
```

Comment:
I think checking question type explicitly is better, because the type can
be assignment upload or in future fill in the blanks. So, this may break.